projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -1,11 +1,11 @@
 import { Component } from '@angular/core';
 import {
-  IConfig, IPageViewTelemetry, IAppInsights, PropertiesPluginIdentifier, Util
+    IConfig, IPageViewTelemetry, IAppInsights, PropertiesPluginIdentifier, Util
```

Comment:
Most of the changes here are whitespace only -- properly aligning the entire file to the same indent

Reply:
You can hide the whitespace changes using the gear drop down above (the one next to the "Jump to" to make life a LOT easier to read this 
![image](https://user-images.githubusercontent.com/54870357/123461761-51de3c00-d59e-11eb-8dea-7ca084500994.png)